urdfenvs/urdfCommon/differential_drive_robot.py

```diff
@@ -99,7 +109,7 @@ def get_observation_space(self) -> gym.spaces.Dict:
         `velocity` the concatenated velocities of joints in their local configuration space.
         `forward_velocity` the forward velocity of the robot.
         """
-
+        self.preload()
```

Comment:
The preloading does not relate to the observation space so it should not be called inside this particular function but rather in either the constructor or the reset function.